package/sap-installation-wizard.changes

```diff
@@ -1,3 +1,32 @@
+-------------------------------------------------------------------
+Thu Jun 15 14:05:22 UTC 2023 - Peter Varkoly <varkoly@suse.com>
+
+- Enable sapconf to apply the required system settings by start.
+- 4.4.8
+
+-------------------------------------------------------------------
+Wed Jun  7 09:43:29 UTC 2023 - Peter Varkoly <varkoly@suse.com>
+
+- Remove all depreceted unified_installer stuff.
+- sap-installation-wizard can not find the installer on B1 image
+  (bsc#1212097) Fix the order of finding the installer.
+- Add some Business One specific tunig stuff  which was earlier integrated into the preinstalled image.
```

Comment:
Typo: tunig

Reply:
Why 4 different changelog entries? Why 2 version bumps? I would expect a single entry and version bump.

Reply:
Because I've already submitted this package internal int SUSE:SLE-15-SP4:Update:SAPBOne. It was neccessary to react to test results. It is a new product new way ...